packages/contracts/test/token/erc20/governance-erc20.ts

```diff
@@ -0,0 +1,221 @@
+
+      await expect(token.initialize(...defaultInitData)).to.be.revertedWith(
+        OZ_ERRORS.ALREADY_INITIALIZED
+      );
+    });
+
+    it('sets the token name and symbol', async () => {
+      token = await GovernanceERC20.deploy(...defaultInitData);
+
+      expect(await token.name()).to.eq(governanceERC20Name);
+      expect(await token.symbol()).to.eq(governanceERC20Symbol);
+    });
+
+    it('sets the managing DAO ', async () => {
+      token = await GovernanceERC20.deploy(...defaultInitData);
+      expect(await token.getDAO()).to.eq(dao.address);
+    });
+
+    it('reverts if the `receivers` and `amounts` array lengths in the mint settings mismatch', async () => {
+      const receivers = [signers[0].address];
+      const amounts = [123, 456];
+      await expect(
+        GovernanceERC20.deploy(
+          dao.address,
+          governanceERC20Name,
+          governanceERC20Symbol,
+          {receivers: receivers, amounts: amounts}
+        )
+      )
+        .to.be.revertedWithCustomError(token, 'MintSettingsArrayLengthMismatch')
+        .withArgs(receivers.length, amounts.length);
+    });
+  });
+
+  describe('supportsInterface:', async () => {
+    it('it supports all inherited interfaces', async () => {
+      token = await GovernanceERC20.deploy(...defaultInitData);
+
+      await Promise.all(
+        [
+          'IERC165Upgradeable',
+          'IERC20Upgradeable',
+          'IERC20PermitUpgradeable',
+          'IVotesUpgradeable',
+          'IERC20MintableUpgradeable',
+        ].map(async interfaceName => {
+          const iface = new ethers.utils.Interface(
+            // @ts-ignore
+            (await hre.artifacts.readArtifact(interfaceName)).abi
+          );
+          expect(await token.supportsInterface(getInterfaceID(iface))).to.be
+            .true;
+        })
+      );
+      // We must check `IERC20MetadataUpgradeable` separately as it inherits from `IERC20Upgradeable` and we cannot get the isolated ABI.
+      const ierc20MetadataInterface = new ethers.utils.Interface([
+        'function name()',
+        'function symbol()',
+        'function decimals()',
+      ]);
+      expect(
+        await token.supportsInterface(getInterfaceID(ierc20MetadataInterface))
+      ).to.be.true;
+    });
+  });
+
+  describe('mint:', async () => {
+    beforeEach(async function () {
+      token = await GovernanceERC20.deploy(...defaultInitData);
+    });
+
+    it('reverts if the `MINT_PERMISSION_ID` permission is missing', async () => {
+      await expect(token.mint(signers[0].address, 123))
+        .to.be.revertedWithCustomError(token, 'DaoUnauthorized')
+        .withArgs(
+          dao.address,
+          token.address,
+          token.address,
+          signers[0].address,
+          MINT_PERMISSION_ID
+        );
+    });
+
+    it('mints tokens if the caller has the `mintPermission`', async () => {
+      dao.grant(token.address, signers[0].address, MINT_PERMISSION_ID);
```

Comment:
missing `await`.

Reply:
Fixed in 6bc76c04704702f2e4765f21971a9f16a5b433c6